src/lib.rs

```diff
@@ -33,6 +33,8 @@ pub use standard_out_api::parse_command_string;
 pub use standard_out_api::ConsoleCommand;
 pub use writer_api::WriterCommand;
 
+pub const CURRENT_WORKING_DIRECTORY: Option<String> = None;
```

Comment:
How does this get set? Through an environment variable?

Reply:
It doesn't. None gets pushed through all the way to `std::process::Command` which defaults to current working directory.

I'm not so much a fan of the current solution or this hanging None. It doesn't feel elegant. I'm willing to discuss solutions but Its currently breaking Theorem so I need to push a quick fix out to get people back to productivity

Reply:
@kevgo basically, the issue is `origins-cli` is trying to run a script with a relative path: say `scripts/setup` from a different directory. However, in the older version, currant checks to see if `scripts/setup` exists which of course, it doesn't. At least until you tell it that it needs to check in the directory `XYZ`. But, the `cur_dir` option can only be set _after_ the CommandNotFound checks.

The current solution is to force the user to provide the working directory in command construction. But that means that a fairly common case (not changing the working directory) requires the user to stick a bunch of None's in the constructor which doesn't feel so elegant to me.

Reply:
You can't actually pass None here because it has generic type (`Option<Into<PathBuf>>`). So rust cannot infer the type of None. Here, the constant is added with an explicit type so people can use the constant without having to do something silly like: 
```
let cwd: Option<String> = None;
let handle = Runner::new()
  .ConsoleCommand::from_str_in_path("cmd2", "echo runs in given directory", cwd);
```

Reply:
I see, so that one can provide `currant::CURRENT_WORKING_DIR` as the value for the current directory instead of saying `None`. That's semantically more meaningful. A possible challenge with this approach is that one has to remember that this constant exists in order to use it. Would an enum with better named variants guide the user better here?

```rs
enum WorkingDir {
  Current,
  Custom(PathBuf)
}